torchtext/experimental/datasets/raw/multi30k.py

```diff
@@ -0,0 +1,526 @@
+                'MD5': 'cbf5bfc2147706f228d288e1b18bf4af',
+                'NUM_LINES': 29000
+            },
+            'train.2': {
+                'URL':
+                    'https://raw.githubusercontent.com/multi30k/dataset/master/'
+                    'data/task2/raw/train.2.en.gz',
+                'MD5': 'bdfe4222f4692ccaa1e3389460f0890e',
+                'NUM_LINES': 29000
+            },
+            'train.3': {
+                'URL':
+                    'https://raw.githubusercontent.com/multi30k/dataset/master/'
+                    'data/task2/raw/train.3.en.gz',
+                'MD5': '0e1ee2b4145795bd180b193424db204b',
+                'NUM_LINES': 29000
+            },
+            'train.4': {
+                'URL':
+                    'https://raw.githubusercontent.com/multi30k/dataset/master/'
+                    'data/task2/raw/train.4.en.gz',
+                'MD5': '1cff688d1aadef7fdb22e9ad27d6fd2c',
+                'NUM_LINES': 29000
+            },
+            'train.5': {
+                'URL':
+                    'https://raw.githubusercontent.com/multi30k/dataset/master/'
+                    'data/task2/raw/train.5.en.gz',
+                'MD5': '3e10289959d0059952511c31df3c7550',
+                'NUM_LINES': 29000
+            },
+            'val.1': {
+                'URL':
+                    'https://raw.githubusercontent.com/multi30k/dataset/master/'
+                    'data/task2/raw/val.1.en.gz',
+                'MD5': 'df57faf5f00d434d2559c021ef55f1aa',
+                'NUM_LINES': 1014
+            },
+            'val.2': {
+                'URL':
+                    'https://raw.githubusercontent.com/multi30k/dataset/master/'
+                    'data/task2/raw/val.2.en.gz',
+                'MD5': '9077a5127480cc799116384de501bd70',
+                'NUM_LINES': 1014
+            },
+            'val.3': {
+                'URL':
+                    'https://raw.githubusercontent.com/multi30k/dataset/master/'
+                    'data/task2/raw/val.3.en.gz',
+                'MD5': 'c1f697c3b6dfb7305349db34e26b45fc',
+                'NUM_LINES': 1014
+            },
+            'val.4': {
+                'URL':
+                    'https://raw.githubusercontent.com/multi30k/dataset/master/'
+                    'data/task2/raw/val.4.en.gz',
+                'MD5': 'acb5ea26a577ceccfae6337181c31716',
+                'NUM_LINES': 1014
+            },
+            'val.5': {
+                'URL':
+                    'https://raw.githubusercontent.com/multi30k/dataset/master/'
+                    'data/task2/raw/val.5.en.gz',
+                'MD5': '680816e0938fea5cf5331444bc09a4cf',
+                'NUM_LINES': 1014
+            }
+        }
+    }
+}
+
+
+URL = {'train': [], 'valid': [], 'test': []}
+MD5 = {'train': [], 'valid': [], 'test': []}
+NUM_LINES = {'train': [], 'valid': [], 'test': []}
+
+for task in SUPPORTED_DATASETS.keys():
```

Comment:
nit: the "in" operator should return a key by default, so I don't think you need .keys() here